src/diffusers/models/unet_2d_condition.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         norm_eps=1e-5,
```

Comment:
```suggestion
        norm_eps: float =1e-5,
```

Could you give all those values some type hints as well? :-)